setup.cfg

```diff
@@ -1,3 +1,16 @@
 [bdist_wheel]
 # code is written to work on both Python 2 and Python 3
-universal=1
+universal = 1
+
+[pep8]
+max-line-length = 119
+# Ignore E127: checked by pylint
+# E127 continuation line over-indented for visual indent
+ignore = E127
+exclude = migrations,setup.py
+
+[pydocstyle]
+match-dir = (?!tests|migrations|\.).*
+match = (?!setup).*.py
+add-select = D203,D404
```

Comment:
Why enable `D203` and disable `D211`?

Pydocstyle changed the default behaviour in [Oct 2015](https://pydocstyle.readthedocs.io/en/latest/release_notes.html#october-9th-2015) due to [Guido van Rossum's change to the official PEP257](https://hg.python.org/peps/rev/9b715d8246db). Guido changed this since the ["previous version didn't reflect the common practice, including the Python standard library"](http://raspberry-python.blogspot.si/2015/01/significant-pep257-change.html?showComment=1420601718075#c4025816682001290873).